bindings/src/main/java/io/pravega/storage/hdfs/HDFSStorage.java

```diff
@@ -561,6 +624,26 @@ private Path getSealedFilePath(String segmentName) {
      * @throws IOException If an exception occurred.
      */
     private FileStatus findStatusForSegment(String segmentName, boolean enforceExistence) throws IOException {
+        FileStatus[] statuses = findAllStatusForSegment(segmentName, enforceExistence);
+        if (null == statuses) {
+            return null;
+        }
+        if (statuses.length > 1) {
+            throw new IllegalArgumentException("More than one file");
+        }
+
+        return statuses[statuses.length -1];
```

Comment:
So, here the method will always return `statuses[0]`, right? If so, maybe `0 ` is more direct than using as index `statuses.length-1`.

Reply:
Agree .